melodic/distribution.yaml

```diff
@@ -12177,6 +12177,17 @@ repositories:
       url: https://github.com/gbiggs/rtsprofile.git
       version: master
     status: maintained
+  ruckig:
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/pantor/ruckig-release.git
+      version: 0.3.3-1
+    source:
+      type: git
+      url: https://github.com/pantor/ruckig.git
```

Comment:
The [LICENSE](https://github.com/pantor/ruckig/blob/master/LICENSE) file seems to contain both the MIT and BSD licenses, but the [package.xml](https://github.com/pantor/ruckig/blob/master/package.xml) only lists MIT.  If BSD code is used in this package, it should have a second `<license>` tag in the `package.xml`.

Reply:
Ruckig is only licensed under MIT, the BSD text was just a leftover... I've removed that now, thanks for pointing out!